### PR TITLE
Adding missing feature descriptions - Batch 11

### PR DESCRIPTION
**Pull request by captainbrosset**

### CODE REVIEW

File: feature-group-definitions/web-animations.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 name: Web animations
+description: The Web Animations API allows to animate, and synchronize the animations of DOM elements.
```
Comment (foolip) [inline feature-group-definitions/web-animations.yml:2]: This API has multiple entry points but I think it's fair to call the `animate()` method the main one. However, I really don't think the feature would be generally recognized by the name "animate", so I guess I'll have to suggest this:

```suggestion
name: Web animations
description: The web animations API allows you to animate and synchronize the animations of DOM elements.
```

File: feature-group-definitions/webp.yml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 name: WebP
+description: The WebP image format is a raster graphics file format that supports animation, alpha transparency, and lossy and lossless compression.
```
Comment (foolip) [inline feature-group-definitions/webp.yml:2]:
```suggestion
description: The WebP image format is a raster graphics file format that supports animation, alpha transparency, and lossy as well as lossless compression.
```

File: feature-group-definitions/webtransport.yml [hunk 3]
```diff
@@ -1,4 +1,5 @@
 name: WebTransport
+description: The WebTransport API transmits data between a client and a server, by using the HTTP/3 protocol.
```
Comment (foolip) [inline feature-group-definitions/webtransport.yml:2]:
```suggestion
description: The `WebTransport` API transmits data between a client and a server, by using the HTTP/3 protocol.
```

**Review by foolip — COMMENTED** [review]
Yay, all of the Web Things making naming/descriptions hard :)

**Comment by foolip** [comment]
@captainbrosset I fixed some "boring" errors, but can you split out `:where()` so I just don't delete it from under you?

File: feature-group-definitions/where.yml [hunk 4]
```diff
@@ -1,4 +1,5 @@
 name: ":where()"
+description: The `:where()` CSS pseudo-class function takes a selector list as its argument, and selects any element that can be selected by one of the selectors in that list. It is functionally equivalent to the selectors in the list, but doesn't affect the CSS rule specificity.
```
[the remark this answers is not in the document]
Comment (captainbrosset) [reply feature-group-definitions/where.yml:2]: See #862.

**Review by foolip — APPROVED** [review]

**Comment by foolip** [comment]
In the interest of getting https://github.com/web-platform-dx/web-features/pull/761 done ASAP I'm going to merge this and make a list of things that @ddbeck might want to review post facto and send tweaks for.

**Pull request merged**